src/odemis/gui/comp/canvas.py

```diff
@@ -851,7 +851,6 @@ def set_images(self, im_args):
 
                     for tile_col in im:
                         for tile in tile_col:
-                            tile.metadata['dc_center'] = tile.metadata.get(model.MD_POS, w_pos)
```

Comment:
That commit can be merged with commit "Use the shape and center of the image composed from the tiles..."